composer.json

```diff
@@ -63,7 +63,8 @@
         "nyholm/psr7-server": "1.0.1",
         "lcobucci/jwt": "dev-342php8 as 3.4.9",
         "psr/log": "1.1.3",
-        "monolog/monolog": "2.1.1"
+        "monolog/monolog": "2.1.1",
+        "squizlabs/php_codesniffer": "3.6.0"
```

Comment:
Shouldn't this be in the require-dev section?  Also is this a requirement for your code to have this package or was this left in accidentally?

Reply:
No clue how that got there.

Reply:
I removed the squizlabs/php_codesniffer